migration/identity-migration/src/main/java/io/camunda/migration/identity/RoleMigrationHandler.java

```diff
@@ -23,58 +20,39 @@
 public class RoleMigrationHandler extends MigrationHandler<Role> {
   private static final Logger LOG = LoggerFactory.getLogger(RoleMigrationHandler.class);
   private final RoleServices roleServices;
-  private final AuthorizationServices authorizationServices;
-  private final ManagementIdentityClient managementIdentityClient;
-  private final ManagementIdentityTransformer managementIdentityTransformer;
+  private final List<CreateRoleRequest> roles =
```

Comment:
💭 should we already call that one `SaaSRoleMigrationHandler`? But I'm also fine to revisist that once we implement the SM handler(s) logic.